src/components/Sidebar/index.jsx

```diff
@@ -51,7 +51,7 @@ export default class Sidebar extends Component {
 
     return (
       <List disablePadding>
-        {Object.keys(BUGZILLA_LANGUAGES).map(language => (
+        {['Swift', ...Object.keys(BUGZILLA_LANGUAGES)].map(language => (
```

Comment:
Why not just include `Swift` in BUGZILLA_LANGUAGES?

Reply:
it does not seem to have the mapping in the whiteboards

Reply:
Ah, in that case, add a comment to make it more clear for someone else reading the code.